sql/codegen.go

```diff
@@ -238,39 +252,70 @@ for name in feature_column_names:
 Y = columns[field_names.index("{{.Y.Name}}")]
 {{- end}}
 
-classifier = tf.estimator.{{.Estimator}}(
-    feature_columns=feature_columns,{{range $key, $value := .Attrs}}
-    {{$key}} = {{$value}},{{end}}
-    model_dir = "{{.Save}}")
+classifier = {{.Estimator}}(
+	feature_columns=feature_columns,{{if .SelfDefined}}{{range $key, $value := .Attrs}}
+	{{$key}} = {{$value}},{{end}})
```

Comment:
Correct me if I am wrong, both self-defined and premade models need `{{range $key, $value := .Attrs}}{{$key}} = {{$value}},{{end}}`, is it possible to combine them?

Reply:
Done